requirements-app.txt

```diff
@@ -2,16 +2,18 @@
 # with package version changes made in requirements-app.txt
 
 aws-wsgi==0.2.7
-boto==2.49.0
+boto3==1.17.58
+botocore==1.20.58
 cffi==1.14.5
-celery[sqs]==5.0.5
+celery[sqs]==5.1.2
 docopt==0.6.2
 fido2==0.9.1
 Flask-Bcrypt==0.7.1
 flask-marshmallow==0.14.0
 Flask-Migrate==2.7.0
 git+https://github.com/mitsuhiko/flask-sqlalchemy.git@500e732dd1b975a56ab06a46bd1a20a21e682262#egg=Flask-SQLAlchemy==2.3.2.dev20190108
-Flask==1.1.2
+# Flask-SQLAlchemy==2.5.1 # TODO: Update to latest soon!
```

Comment:
I did some many library upgrades, I wanted to wait for that one because it could have an impact.